nengo_gui/guibackend.py

```diff
@@ -295,7 +295,7 @@ def get_session(self):
         return session
 
     def log_message(self, format, *args):
-        logger.info(format, *args)
+        logger.debug(format, *args)
```

Comment:
This function is called from the web server base class and is called once for each incoming request.